exchanges/exchange.go

```diff
@@ -1943,3 +1945,30 @@ func (b *Base) GetTradingRequirements() protocol.TradingRequirements {
 	}
 	return b.Features.TradingRequirements
 }
+
+// FetchTicker returns the ticker for a currency pair and asset type
+// associated with the exchange.
+// NOTE: UpdateTicker (or if supported UpdateTickers) method must be
+// called first to update the ticker map.
+func (b *Base) FetchTicker(_ context.Context, p currency.Pair, assetType asset.Item) (*ticker.Price, error) {
```

Comment:
How about a name change of these from FetchX to FetchCachedX (Or FetchTickerCatched)? Therefore its purpose is explicit in its name in case people overlook these notes

Reply:
👍 🌈

Reply:
[done](https://github.com/thrasher-corp/gocryptotrader/pull/1440/commits/54db17d784d3e1289ad5c3c5626f59007b6b5e8e)